### DISCUSSION TITLE
playTrack() volume property and filter volume is inconsistent

### DISCUSSION THREAD

**Discussion by Deivu** [lavalink-devs/Lavalink#883] [Q&A]
## Description
The volume value on playTrack() is not in the same scale of filter volume. This causes a lot of confusion on setting values due to the volume value on play track is apparently on scale of 0-1000 while filter volume is on 0.1-5. I suggest making both use the same value to avoid confusion

**Comment by topi314** [comment]
While I see where you are coming from, they are not really the same.
Filters volume takes x buffer size to apply while player volume applies instantly.

The only usage for filter volume would be, if you have something like a nightcore filter where you would adjust the volume. In this case you want to actually use both then, and not just one.

Client libraries should have propper documentation & ideally types for these.

**Reply by Deivu** [reply]
Thanks for the answer, I do remember not seeing the values on the implementation way back before which caused confusions, and seeing now it's properly documented there. One thing though, probably put what is the differences in the impl. guidelines to help users know?

**Reply by topi314** [reply]
feel free to pr a change for this
